c7n/resources/rds.py

```diff
@@ -1931,3 +1931,55 @@ def process(self, resources, event=None):
                 r['c7n:Engine'] = v
                 matched.append(r)
         return matched
+
+
+@filters.register('count')
```

Comment:
let's make this a little more clear in policies, by renaming the filter to 'snapshot-count' .. count is a little ambigious.